entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java

```diff
@@ -124,7 +124,7 @@ public ResponseEntity<String> disableEntity(
           String profile,
       HttpServletRequest request)
       throws HttpException, EuropeanaApiException {
-    if (emConfig.isAuthEnabled()) {
+    if (emConfig.isAuthWriteEnabled()) {
```

Comment:
the dupplications of these checks must be removed by overwriting the verifyReadAccess and verifyWriteAccess methods in the BaseRest class https://github.com/europeana/entity-management/blob/27eef7efbf7db0b6f7e7b0733be45316091f4a3d/entity-management-web/src/main/java/eu/europeana/entitymanagement/web/BaseRest.java#L53